packages/components/src/pages/Quickstart.vue

```diff
@@ -0,0 +1,571 @@
+            >
+              {{ languages[lang] }}
+            </option>
+          </select>
+        </div>
+        <div class="qs-step__block" v-if="!step1Completed">
+          <p>
+            This will add the following snippet to the top of your Gemfile and
+            run bundle to install the AppMap gem.
+          </p>
+          <code class="qs-code" @click="select"
+            >gem "appmap", "= 0.53.0", :groups => [:development, :test]</code
+          >
+          <button class="qs-button" v-if="!isActionRunning" @click="runAction">
+            Install the AppMap agent
+          </button>
+          <div class="qs-loader" v-if="isActionRunning">
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+          </div>
+        </div>
+        <div class="qs-step__success" v-if="step1Completed">
+          <span class="qs-step__success-title">
+            <SuccessIcon class="qs-step__success-icon" />
+            Agent installed
+          </span>
+          <button
+            type="button"
+            class="qs-step__success-next-step qs-button"
+            @click="nextStep"
+          >
+            Next step : Configure AppMap ->
+          </button>
+        </div>
+      </section>
+      <section class="qs-step" v-if="currentStep === 2">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Configure AppMap</h1>
+        </div>
+        <div class="qs-step__block" v-if="!step2Completed">
+          <p>
+            This will create an <a href="#">appmap.yml</a> config file in the
+            root directory of your project. This will tell AppMap what code to
+            record. You can run these defaults or add more packages, gems, or
+            specific functions. You can edit this file at any time.
+          </p>
+          <!-- prettier-ignore -->
+          <code class="qs-code" @click="select"
+            ># AppMap RUBY template
+# 'name' should generally be the same as the code repo name.
+ name: my_project
+ packages:
+ - path: app/controllers
+ - path: app/models
+ # Include the gems that you want to see in the dependency maps.
+ # These are just examples.
+ - gem: activerecord
+ - gem: devise</code
+          >
+          <button class="qs-button" v-if="!isActionRunning" @click="runAction">
+            Create appmap.yml config file
+          </button>
+          <div class="qs-loader" v-if="isActionRunning">
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+          </div>
+        </div>
+        <div class="qs-step__success" v-if="step2Completed">
+          <span class="qs-step__success-title">
+            <SuccessIcon class="qs-step__success-icon" />
+            AppMap configuration file created
+          </span>
+          <span class="qs-step__success-subtitle">appmap.yml</span>
```

Comment:
Can this also be a link that emits an event upon being clicked? Then we can listen for the event from the code editor and open the file.